pymathics/graph/centralities.py

```diff
@@ -71,30 +69,31 @@ class BetweennessCentrality(_Centrality):
     :Betweenness centrality:
     https://en.wikipedia.org/wiki/Betweenness_centrality</url> (<url>
     :NetworkX:
-    https://networkx.org/documentation/networkx-2.8.8/reference/algorithms/generated/\
-    networkx.algorithms.centrality.betweenness_centrality.html</url>,\
+    https://networkx.org/documentation/networkx-2.8.8/reference/algorithms/generated/networkx.algorithms.centrality.betweenness_centrality.html</url>,\
```

Comment:
Note - no space used here. I find it useful to look at the docs after modifying them. It catches stuff like this.